packages/react/src/ActionList/List.tsx

```diff
@@ -33,15 +33,20 @@ export const List = React.forwardRef<HTMLUListElement, ActionListProps>(
 
     /** if list is inside a Menu, it will get a role from the Menu */
     const {
-      listRole,
+      listRole: listRoleFromContainer,
       listLabelledBy,
       selectionVariant: containerSelectionVariant, // TODO: Remove after DropdownMenu2 deprecation
-      enableFocusZone,
+      enableFocusZone: enableFocusZoneFromContainer,
     } = React.useContext(ActionListContainerContext)
 
     const ariaLabelledBy = slots.heading ? slots.heading.props.id ?? headingId : listLabelledBy
-
+    const listRole = role || listRoleFromContainer
     const listRef = useProvidedRefOrCreate(forwardedRef as React.RefObject<HTMLUListElement>)
+
+    let enableFocusZone = false
```

Comment:
If there's an existing focus zone being used for an `ActionList`, would the one being used here replace it? I saw an instance where there was an `ActionList` inside of an `AnchoredOverlay` that had an implicit focus zone attached: 

```tsx
<AnchoredOverlay
  renderAnchor={props => <Button {...props}>Button</Button>}
  focusZoneSettings={{
    focusOutBehavior: 'wrap',
  }}
>
  <ActionList role="listbox">
    <ActionList.Item>Item 1</ActionList.Item>
    <ActionList.Item>Item 2</ActionList.Item>
  </ActionList>
</AnchoredOverlay>
```
I'm wondering if this would be an issue or not 🤔. I think the existing cases are rare, so doesn't seem like it would be.

Reply:
focus zone can only be added, not removed. So, if the parent AnchoredOverlay has a focus zone, that would work on all it's children. Setting it to false here would not disable to parent. (ActionMenu, Autocomplete, etc. also use AnchoredOverlay)